packages/generator-single-spa/src/react/generator-single-spa-react.js

```diff
@@ -1,26 +1,36 @@
 const Generator = require('yeoman-generator')
 const ejs = require('ejs')
 const fs = require('fs').promises
+const chalk = require('chalk')
 
 module.exports = class SingleSpaReactGenerator extends Generator {
+  constructor(args, opts) {
+    super(args, opts)
+
+    this.option("packageManager", {
+      type: String
+    })
+  }
   async createPackageJson() {
-    const answers = await this.prompt([
-      {
-        type: "list",
-        name: "packageManager",
-        message: "Which package manager do you want to use?",
-        choices: [
-          "yarn",
-          "npm",
-        ]
-      }
-    ])
+    this.packageManager = this.options.packageManager
 
-    this.packageManager = answers.packageManager
+    if (!this.packageManager) {
+      this.packageManager = (await this.prompt([
+        {
+          type: "list",
+          name: "packageManager",
+          message: "Which package manager do you want to use?",
+          choices: [
+            "yarn",
+            "npm",
+          ]
+        }
+      ])).packageManager
+    }
```

Comment:
I like this change, if I understand correctly this will allow them to pass in the package manager during the create process.

Reply:
Yeah it means they can pass it in as a CLI argument:

```sh
npm init single-spa --packageManager yarn
```